newsAPI.js

```diff
@@ -1,107 +1,48 @@
-
-$(document).ready(()=>{
-	getNews();
-
-	function getNews()
-	{
-		let endPoint = "https://newsapi.org/v2/top-headlines";
-		let apiKey = "0873dd38116a4b1d9db9c7f2d99754a7";
```

Comment:
Will our app pull information from newsapi.org without lines 7 and 8?

Reply:
that's all in the backend.  you do need to npm install the news api node mod though.

Reply:
ok, doing it now.